packages/client/scripts/codegen.cjs

```diff
@@ -236,4 +237,28 @@ const mergeArrays = (arr1, arr2) => {
   return Array.from(map.values());
 };
 
+async function fixProtobufjsImports(directory) {
+  const entries = await fs.readdir(directory, { withFileTypes: true });
+
+  for (const entry of entries) {
+    const fullPath = path.join(directory, entry.name);
+    if (entry.isDirectory()) {
+      await fixProtobufjsImports(fullPath);
+    } else if (entry.isFile() && entry.name.endsWith(".ts")) {
+      let content = await fs.readFile(fullPath, "utf-8");
+      let updated = content;
+
+      // Replace import * as x from 'protobufjs/minimal'; with import x from 'protobufjs/minimal.js'
+      updated = updated.replace(
+        /import\s+\*\s+as\s+(\w+)\s+from\s+['"]protobufjs\/minimal(?:\.js)?['"];/g,
+        'import $1 from "protobufjs/minimal.js";'
+      );
+
+      if (updated !== content) {
+        await fs.writeFile(fullPath, updated, "utf-8");
+      }
+    }
+  }
+}
+
```

Comment:
this function manually patches protobufjsImports to follow esm import convention, after telescope generates them